recipes/pyscenic/meta.yaml

```diff
@@ -0,0 +1,87 @@
+{% set name = "pyscenic" %}
+{% set version = "0.12.1" %}
+{% set sha256 = "ae8fafa707d2578ffe08f9eed85f14a4cd9e1b53d57217420e2e956f0a8ddba2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  - url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+    sha256: "{{ sha256 }}"
+    folder: pyscenic-0.12.1
+  - url: "https://files.pythonhosted.org/packages/source/a/arboreto/arboreto-0.1.6.tar.gz"
+    sha256: "32fdac5e8a3e0ef2e196b5827f067d815ac4e689d2fca0dc437f42abdeeb89ab"
+    folder: arboreto-0.1.6

```

Comment:
@LiliyaBioinf arboreto is available from conda, and as such should be used as a conda dependency. Can you remove it from the sources and add under requirements.run please? Can you do the same with any of the above sources that might also be in conda (usually searching for conda + name of the package should allow you to find them if the exists as conda packages). Thanks!

Reply:
https://anaconda.org/bioconda/arboreto

Reply:
and of course, if added to the requirements, then shouldn't go in the `build.sh`.

Reply:
Looks like there was some discussion a while back to add multiprocessing_on_dill to [conda-forge ](https://github.com/conda-forge/staged-recipes/pull/6199), but the upstream maintainers did not approve the [PR to add the license](https://github.com/sixty-north/multiprocessing_on_dill/issues/1). That would need to be resolved in some way so it can be added to conda-forge.

Reply:
maybe in the meantime multiprocessing_on_dill can be installed through pip so that the community can benefit soon from this recipe.